### DISCUSSION TITLE
Show future days

### DISCUSSION THREAD

**Discussion by yea-hung** [iSoron/uhabits#1998] [Feature Requests]
I would love a setting that would allow for the display of future days, as well as past days. For example, if today is Monday, it would be nice to have a setting that would display the Tuesday (+1) and Wednesday (+2), as well as the Monday and Sunday (-1). This would help me prepare for upcoming habits, such as weekly habits.

**Comment by ezio-melotti** [comment]
See these discussions:
* https://github.com/iSoron/uhabits/discussions/372
* https://github.com/iSoron/uhabits/discussions/1160
* https://github.com/iSoron/uhabits/discussions/2042
